content/docs/prerequisites/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Prerequisites"
+linkTitle: "Prerequisites"
+description: Prerequisites for CSM 
+weight: 1
+---
+
+## Supported Storage Platforms
+
+{{<table "table table-striped table-bordered table-sm">}}
+| Platform      | Version          | OS Dependencies          |
+|---------------|:----------------:|:------------------------:|
+| PowerMax    | PowerMax 2500/8500 PowerMaxOS 10 (6079)<br>PowerMaxOS 10.0.1 (6079)<br>PowerMaxOS 10.1 (6079)<br>PowerMax 2000/8000 - 5978.711.xxx<br>5978.479.xxx<br>Unisphere 10.0,10.0.1,10.1 | iscsi-initiator-utils<br>multipathd or powerpath<br>nvme-cli<br>nfs-utils |
+| PowerFlex | 3.6.x, 4.0.x, 4.5.x | [SDC](https://www.dell.com/support/home/en-us/product-support/product/scaleio/drivers)|
+| Unity XT | 5.1.x, 5.2.x, 5.3.0 | iscsi-initiator-utils<br>multipathd<br>nfs-utils |
+| PowerScale | OneFS 9.3, 9.4, 9.5.0.x (x >= 5) | nfs-utils |
+| PowerStore  | 3.0, 3.2, 3.5 | iscsi-initiator-utils<br>multipathd<br>nvme-cli<br>nfs-utils |
+| ObjectScale | 1.2.x | - |
+{{</table>}}
+
+> Notes:
+> * The required OS dependencies are only for the protocol needed (e.g. if NVMe isn't the storage access protocol then nvme-cli is not required).
+> * [CSM Authorization](../authorization/) only supports Kubernetes and the following Storage Platforms: PowerMax, PowerFlex, and PowerScale.
+> * [CSM Observability](../observability/) does not support the Unity XT Storage Platform.
+> * [CSM Replication](../replication/) does not support the Unity XT Storage Platform or File Replication for PowerMax.
+
+## Supported Container Orchestrator Platforms
+
+{{<table "table table-striped table-bordered table-sm">}}
+| Platform      | Version          |
+|---------------|:----------------:|
+| Kubernetes    | 1.27, 1.28, 1.29 |
+| Red Hat OpenShift | 4.13, 4.14 |
+| Mirantis Kubernetes Engine | 3.7.x |
+| Google Anthos |        1.15      |
+| Rancher Kubernetes Engine | 1.4.x|
+| Amazon Elastic Kubernetes Service | ?? |
```

Comment:
This is Amazon Elastic Kubernetes Anywhere,not Amazon EKS

Reply:
Thanks.  Removed this from the support matrix since Amazon Elastic Kubernetes Anywhere is connected to a backed Kubernetes version